.artifactignore

```diff
@@ -7,3 +7,8 @@ start-docker.sh
 update-version.sh
 Dockerfile
 /test
+cdk.out
+cdk.json
+cdk.context.json
+jest.config.js
+node_modules
```

Comment:
It's better to add a new empty line at the end of files.
This can generally be configured in your favorite editor on save.

Reply:
done